src/lib/CLValue/constants.ts

```diff
@@ -71,6 +71,8 @@ export enum CLTypeTag {
   PublicKey = 22
 }
 
+export const ALIAS_ACCOUNT_HASH_TO_BYTE_ARRAY = -1;
```

Comment:
Can you please comment this one ? what does this mean exactly to receive -1 ?

Where do you use it please ? Either github tricks me, either it is not used.

Reply:
it was a temp thing, if you look at the previous implementation `AccountHash` was only type used internally which was not possible to serialize. Now it's not used anymore, I removed it, thanks!